data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -41,16 +43,20 @@ def __init__(self):
 
         self.TILES_SCORE_COLUMNS = [
             "GEOID10",
+            "State Name",
+            "County Name",
+            "Total population",
             "Score E (percentile)",
             "Score E (top 25th percentile)",
-            "GEOID",
-            "State Abbreviation",
-            "County Name",
+            "Poverty (Less than 200% of federal poverty line)",
+            "Percent individuals age 25 or over with less than high school degree",
+            "Linguistic isolation (percent)",
+            "Unemployed civilians (percent)",
+            "Housing burden (percent)",
         ]
```

Comment:
Geoplatform had brought up that the length of the column names themselves can be expensive to write out; might be interesting to see if shrinking the names of the columns has a big impact on tile size?

Reply:
If so, we would need some sort of legend on disk that we could consult, mapping shortened name to expanded, or something like this

Reply:
Per our conversation on zoom, we will punt this as the binaries don't have props like Geojson files do.